docs/Using-Fleet/REST-API.md

```diff
@@ -5321,7 +5329,7 @@ Returns the query specified by ID.
 }
 ```
 
-### Delete query
+### Delete query by name
```

Comment:
Note: If there is a db constriant for uniqueness on name, we'll need to shift this to be unique name for each team.

Reply:
@rachaelshaw was this addressed or discussed at all? Just wanna confirm it's been recognized. We should be explicit somewhere,  maybe in the requirements doc whether we want to make a change to the uniqueness rule...

Reply:
Any updates on this?

Reply:
I think we did address this... Uniqueness should be enforced per team rather than everywhere, since queries now belong to teams. @rachaelshaw, right? 

cc @juan-fdz-hawa @lucasmrod @sharon-fdm for visibility.

Reply:
👍 name needs to be unique among a team's queries but not globally unique.

Reply:
Proposing a clear technical way to say this.
**We need uniqueness on the concatenation of the NAME and TEAM fields.**

Reply:
> name needs to be unique among a team's queries but not globally unique.

Correct, this will be enforced on the db (uniqueness considering the team id too).